su-template/default/module/html/method_summary.erb

```diff
@@ -0,0 +1,16 @@
+<% if method_listing.size > 0 %>
+  <% groups(method_listing) do |list, name| %>
+    <h2>
+      <a name="<%= name.gsub(' ', '_').downcase %>">
```

Comment:
I added this anchor to method and constant summary section. Will push later today.